package.json

```diff
@@ -13,25 +13,27 @@
     "EonEnglish"
   ],
   "scripts": {
+    "dev": "next dev",
     "start": "next start",
     "start:dev": "next dev --turbopack",
     "build": "next build",
     "lint": "next lint"
   },
   "dependencies": {
+    "autoprefixer": "^10.4.20",
+    "next": "^15.1.6",
     "react": "^19.0.0",
-    "react-dom": "^19.0.0",
-    "next": "15.1.3"
+    "react-dom": "^19.0.0"
   },
   "devDependencies": {
-    "typescript": "^5",
+    "@eslint/eslintrc": "^3",
```

Comment:
should be in another PR and attached to another ticket(issue) as there we also need to add custom configuration and standard rules